submissions/maryOkosun/javascript/loop.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Kindly use the same HTML your use for the previous one. No need for creating new HTML page